templates/pages/setup/general/general_setup.html.twig

```diff
@@ -272,6 +272,47 @@
     </script>
 </div>
 
+<div class="hr-text">
+    <i class="ti ti-layout-kanban"></i>
+    <span>{{ __('Project Task State') }}</span>
+</div>
+
+<div class="row ps-4">
+
+    {{ fields.dropdownField(
+        'ProjectState',
+        'projecttask_unstarted',
+        config['projecttask_unstarted']|default('0'),
+        __('Status of unstarted tasks'),
+        inline_field_options|merge({
+            'width': 'auto',
+            'rand': locks_rand,
```

Comment:
Seems to be a variable dedicated to lock fields. Same remark for `projecttask_inprogress` and `projecttask_completed` fields.
```suggestion
```